superset-frontend/src/SqlLab/main.less

```diff
@@ -409,6 +409,10 @@ div.tablePopover {
   padding-right: 8px;
 }
 
+.ResultSetErrorMessage {
```

Comment:
do you mind use "-" not case in css class?

Reply:
my bad, updated!